core/state-db/src/noncanonical.rs

```diff
@@ -270,9 +272,9 @@ impl<BlockHash: Hash, Key: Hash> NonCanonicalOverlay<BlockHash, Key> {
 mod tests {
 	use std::io;
 	use super::NonCanonicalOverlay;
-	use {ChangeSet};
+	use crate::{ChangeSet};
```

Comment:
```suggestion
	use crate::ChangeSet;
```

Reply:
I also prefer to omit braces in such cases, but I don't want to add unnecessary changes.
Fixed.